wiki/en/en-Walk-Through.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: wiki
+title: "Jamulus Walk-Through"
+lang: "en"
+permalink: "/wiki/Walk-Through"
+---
+# Jamulus walk-through
+Let's have a look at what Jamulus looks like and what you should do after having installed the software.
+## First start
+Before you start Jamulus:
+1. Close all other applications
+2. Plugin everything (Ethernet cable, audio gear, etc.)
+
+... to minimise CPU load and to ensure that no other application tries to use your sound card.
+
+You can have a look at [Hardware Setup](Hardware-Setup) for more information on how to set up your hardware. If you need help setting up ASIO4All on Windows, have a look at the [ASIO4All Setup guide on the Windows installation page](Installation-for-Windows#setting-up-asio4all).
+
+### Jamulus main window
+If you open Jamulus, you will see a window which looks like this:
+
+![Jamulus main window](https://user-images.githubusercontent.com/20726856/100796017-4cfa4500-3420-11eb-9d35-aa206d392a5a.png)
+
+### 1. Setting up a profile
+To allow others to identify who you are, you should set your profile. To do so, click on "View" (bar on the top)>"My Profile..."
```

Comment:
Given the apparent nerves around privacy (we don't want to make it sound as if others can find your personal details), perhaps this should be re-cast as: "To let others know who you are, you should set up your profile." (note also "set up").

Reply:
Done